### ISSUE TITLE
Stack trace on blocked update check

### ISSUE THREAD

**Issue by SlimeDog** [Zrips/Residence#763]
Spigot or Paper 1.17.1
CMILib 1.0.2.14
Residence 5.0.0.1

I appreciate the need for the INFO message (at the end) but the SocketTimeoutException should be caught and handled.
```
[15:05:56 WARN]: java.net.SocketTimeoutException: Connection timed out
[15:05:56 WARN]:        at me.lucko.networkinterceptor.interceptors.ProxySelectorInterceptor$LoggingSelector.select(ProxySelectorInterceptor.java:65)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1222)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1110)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1108)
[15:05:56 WARN]:        at java.base/java.security.AccessController.doPrivileged(AccessController.java:691)
[15:05:56 WARN]:        at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:934)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1107)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:175)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1653)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1569)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1567)
[15:05:56 WARN]:        at java.base/java.security.AccessController.doPrivileged(AccessController.java:691)
[15:05:56 WARN]:        at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:934)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1566)
[15:05:56 WARN]:        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
[15:05:56 WARN]:        at net.Zrips.CMILib.Util.CMIVersionChecker.getOfficialVersion(CMIVersionChecker.java:97)
[15:05:56 WARN]:        at net.Zrips.CMILib.Util.CMIVersionChecker.lambda$0(CMIVersionChecker.java:79)
[15:05:56 WARN]:        at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftTask.run(CraftTask.java:101)
[15:05:56 WARN]:        at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54)
[15:05:56 WARN]:        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[15:05:56 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[15:05:56 WARN]:        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[15:05:56 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)
[15:05:56 INFO]: Failed to check for Residence update on spigot web page.
```

**Comment by Zrips** [comment]
it is caught and handled, this simply outputs exact error it produced for simpler debuging. I can simply avoid showing this message to avoid scaring people off. This will be updated with next CMILib release

**Comment by SlimeDog** [comment]
Confirmed fixed in CMILib 1.0.3.4 in combination with Residence 5.0.0.2. Thanks.
